core/renderers/common/constants.js

```diff
@@ -186,6 +186,56 @@ Blockly.blockRendering.ConstantProvider = function() {
   this.disabledPattern_ = null;
 
   /**
+   * Cursor colour.
+   * @type {string}
+   * @package
+   */
+  this.CURSOR_COLOUR = '#cc0a0a';
+
+  /**
+   * Immovable marker colour.
+   * @type {string}
+   * @package
+   */
+  this.MARKER_COLOUR = '#4286f4';
+
+  /**
+   * Width of the horizontal cursor.
+   * @type {number}
+   * @package
+   */
+  this.WS_CURSOR_WIDTH = 100;
+
+  /**
+   * Height of the horizontal cursor.
+   * @type {number}
+   * @package
+   */
+  this.WS_CURSOR_HEIGHT = 5;
+
+  /**
+   * Padding around a stack.
+   * @type {number}
+   * @package
+   */
+  this.STACK_PADDING = 10;
+
+  /**
+   * Padding around a block.
+   * @type {number}
+   * @package
+   */
+  this.BLOCK_PADDING = 2;
```

Comment:
Rename to CURSOR_BLOCK_PADDING.  And the previous one should be CURSOR_STACK_PADDING

Reply:
done.